BUILDING.md

```diff
@@ -527,6 +527,29 @@ $  ./configure --debug --enable-asan && make -j4
 $ make test-only
 ```
 
+#### Speeding up frequent rebuilds when developing
+
+If you plan to frequently rebuild Node.js, especially if using several branches,
+installing `ccache` can help to greatly reduce build times. Setup with:
+```console
+$ sudo apt install ccache
+$ ccache -o cache_dir=<tmp_dir>
+$ ccache -o max_size=5.0G
+$ export CC="ccache gcc"    # add to your .profile
+$ export CXX="ccache g++"   # add to your .profile
+```
```

Comment:
this command set works only for specific variant(s) of UNIX and shell? which is fine, but just adding a line stating so may be useful?